### PR TITLE
Fix display of enums for nullable property

### PR DESCRIPTION
**Pull request by bedfir**
## Subject

Same issue than previously fixed in #8039 

## Changelog

```markdown

### Fixed
- Fix display of enums for nullable property

```

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Regression was introduced here https://github.com/sonata-project/SonataAdminBundle/pull/8070 cc @phansys 

Would be great if one could add a non-regression test

**Comment by phansys** [comment]
>Would be great if one could add a non-regression test

I'll try to take a look this week if it's not too late.
Thanks for pointing.

File: src/Resources/views/CRUD/display_enum.html.twig [hunk 1]
```diff
@@ -10,13 +10,16 @@ file that was distributed with this source code.
 #}
 
 {%- apply spaceless %}
-    {% set value = use_value|default(false) ? value.value : value.name %}
-
-    {% if translation_domain|default(null) is null %}
-        {% set value = value %}
+    {% if value is null %}
```
Comment (phansys) [inline src/Resources/views/CRUD/display_enum.html.twig:13]: In order to cover this case and avoid further regressions, please add a case at [`RenderElementExtensionTest::getRenderListElementTests()`](https://github.com/bedfir/SonataAdminBundle/blob/cf2cc488d6f9836ac3c2cc8e98906142c8aed45b/tests/Twig/Extension/RenderElementExtensionTest.php#L1541):

```php
$elements[] = [
    '<td class="sonata-ba-list-field sonata-ba-list-field-enum" objectId="12345"> &nbsp; </td>',
    FieldDescriptionInterface::TYPE_ENUM,
    null,
    [],
];
```

**Review by phansys — CHANGES_REQUESTED** [review]
Thanks for this contribution @bedfir!

**Review by VincentLanglet — APPROVED** [review]

**Review by phansys — APPROVED** [review]

**Pull request merged**